tests/const.py

```diff
@@ -20,3 +21,7 @@
     "https://raw.githubusercontent.com/"
     "ukgovld/registry-core/master/src/main/vocabs/registryVocab.ttl"
 )
+
+P06_DUMP_FILE = Path(__file__).parent / "data/20230605-P06-dump.ttl"
+
+FAKE_DUMP_FILE = Path(__file__).parent / "data/fake-dump.ttl"
```

Comment:
we are using os.path on some occasions to have the same logic.
I should only adopt one of the two cases I think. 
I personally have no preference whatsoever.

Reply:
I prefer pathlib :)
it is limiting to python3 
but I don´t see that as a real limitation?